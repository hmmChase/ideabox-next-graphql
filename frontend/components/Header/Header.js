import styled from 'styled-components';
import IdeaCardForm from '../IdeaCardForm/IdeaCardForm';

const Header = () => (
  <>
    <StyledHeader>
      <h1>IdeaBox</h1>
      <IdeaCardForm />
    </StyledHeader>
  </>
);

export default Header;

const StyledHeader = styled.header`
  display: flex;
  align-items: center;
  flex-direction: column;
  background-color: #282c34;
  padding: 20px 0;
  h1 {
    margin: 0;
    font-size: 2em;
    color: white;
    margin-bottom: 20px;
    font-family: 'Play', sans-serif;
    color: ${props => props.theme.yellow};
  }
`;
