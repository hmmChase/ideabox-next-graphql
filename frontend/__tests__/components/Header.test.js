import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header/Header';

describe('Header', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = { children: [] };
    wrapper = shallow(<Header {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });
 
  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
