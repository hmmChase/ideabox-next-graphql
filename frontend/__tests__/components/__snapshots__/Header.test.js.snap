// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<Fragment>
  <Header__StyledHeader>
    <h1>
      IdeaBox
    </h1>
    <IdeaCardForm />
  </Header__StyledHeader>
</Fragment>
`;
